packages/rn-tester/RNTester/AppDelegate.mm

```diff
@@ -47,6 +47,9 @@ - (NSDictionary *)prepareInitialProps
 
 - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 {
+#if !USE_HERMES
+  [[RCTBundleURLProvider sharedSettings] setInlineSourceMap:YES];
+#endif
```

Comment:
Any issue with leaving this in here? I'll add an extra note about why it's important in the current state, but I'm also happy to remove it.

Reply:
I think can stay. It is an implementation detail, at the end of the day. Unless it changes the behavior of Hermes, I think it is good.